plugins/storyblok-renderer.ts

```diff
@@ -2,7 +2,7 @@ export default defineNuxtPlugin((useNuxtApp) => {
   /* Resolves the storyblok component */
   // It is needed so that the components are rendered on the server instead of the client.
   function resolveStoryBlokComponent(story) {
-    if (!story) { return null; }
+    if (!story && (story?.component || story?.content?.component)) { return null; }
```

Comment:
This would never be true as story can't be falsey and have component or content.component.

I guess it should be:
`if(!(story?.component || story?.content?.component)) return null`

Reply:
Yes, there is a pair of parentheses missing. Luckily this is a hobby project that doesn't matter and this is a comment on a PR that has been completed.